src/coins.ts

```diff
@@ -27,6 +27,8 @@ export const coins = CoinMap.fromCoins([
   account('teth', 'Testnet Ethereum', Networks.test.kovan, 18, UnderlyingAsset.ETH, ETH_FEATURES),
   account('eos', 'Eos', Networks.main.eos, 4, UnderlyingAsset.EOS),
   account('teos', 'Testnet Eos', Networks.test.eos, 4, UnderlyingAsset.EOS),
+  account('trx', 'Tron', Networks.main.trx, 6, UnderlyingAsset.TRX),
+  account('ttrx', 'Shasta', Networks.test.trx, 6, UnderlyingAsset.TRX),
```

Comment:
Is this the commonly accepted display name for this coin?

Reply:
Renamed to be aligned with the other testnet descriptions